Typo fix

MoSchaeffler opened the pull request:
Fixes https://github.com/MDAnalysis/mdanalysis/issues/4026

Fixed a typo in the documentation of the selection in the example for hydrogen bonds between different group.

Changes made in this Pull Request:
 -  added two ")"

MoSchaeffler committed:
Typo fix

Fixed a typo in the documentation of the selection in the example for hydrogen bonds between different group.

hmacdope reviewed (approved):
LGTM thanks for the fix. 

As you are a first time contributor could you please introduce yourself on the [developer mailing list](https://groups.google.com/g/mdnalysis-devel) so we can get to know you. Please also add yourself to package/AUTHORS as part of this PR. Then I can merge.

orbeckst reviewed (changes requested):
Thank you!
- [ ] add yourself to AUTHORS
- [ ] add your GitHub handle @MoSchaeffler to the CHANGELOG (because you are contributing to the next release)

orbeckst commented:
@MoSchaeffler if you're using the GH webinterface then you have to navigate to AUTHORS and CHANGELOG on your https://github.com/MoSchaeffler/mdanalysis/tree/patch-1 branch. In general, we recommend working on a local checked out repository and then pushing any changes to the branch. See more under [Contributing](https://userguide.mdanalysis.org/stable/contributing.html) in the User Guide.

orbeckst commented:
@MoSchaeffler please address the remaining points or we will close the PR. Thanks!

MoSchaeffler committed:
Update AUTHORS

Addded me (Moritz Schaeffler) to Autors

MoSchaeffler committed:
Update CHANGELOG

added the change by MoSchaeffler to the Changelog, involving issue #4026 and pull request #4029

MoSchaeffler commented:
@orbeckst addressed all remaining points, sorry for the delay!

hmacdope commented:
@MoSchaeffler you will need to fix your merge conflicts with `develop`. Try merging with upstream/develop with `git fetch upstream && git merge upstream/develop` (assuming you have mda remote set up as `upstream`)

MoSchaeffler commented:
Okay, so I merged the develop branch locally and pushed it with my changes to my github branch. Should I open a new pull request or can you merge it like that?

PS: sorry I am github newbie and had to figure out the basics...

MoSchaeffler commented:
Created a new request

orbeckst commented:
I realize now that you created PR #4049 in place of this one. Normally you don't have to do this, you just keep pushing to this branch. We generally prefer you don't replace PRs because it looses the whole history of reviews and comments. But we also understand that git is a difficult tool to learn so we'll continue on PR #4049.

When you ask a question like https://github.com/MDAnalysis/mdanalysis/pull/4029#issuecomment-1450204424 then give us at least a day before you take any major steps like closing a PR. Most of the developers are volunteering their time here and people have jobs so just be a bit patient, please.
